src/aggregations/aggregation-builder.js

```diff
@@ -0,0 +1,64 @@
+import _ from 'lodash'
+import aggregations from './index'
+
+const AggregationBuilder = {
```

Comment:
Not critical but why did you chose a plain old object instead of a es2015 class? `Object.create(X)` seems like a non-standard alternative to `new X()` but maybe I'm missing something.